src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+
+    private static final String OPERATION_BUY = "buy";
+
+    private static final String OPERATION_RESULT = "result";
+
+    private static final String COMA_SEPARATOR = ",";
+
+    private static final String TEXT_OF_EXCEPTION = "File not found";
```

Comment:
we also need to put more details about the **file** to the exception message. You can use '%' for put value later in the methods, or not using constant.